app/controllers/groups_controller.rb

```diff
@@ -3,7 +3,7 @@ class GroupsController < ApplicationController
 
   before_action :authenticate_user!, except: [ :show ]
   before_action :find_group, except: [:index, :new, :create, :feature]
-  before_action :is_group_admin, only: [:update, :destroy, :update]
```

Comment:
Actually I have removed one :update because it was mentioned two times in the before_action callback in groups_controller.rb